docs/public/static/components-gallery/base-theme.css

```diff
@@ -20,157 +20,336 @@
+  --shadow-interactive-switch-track: 0px 1px 1px 0px rgba(0, 0, 0, 0.05) inset;
+  --shadow-interactive-switch-thumb: 0px 1px 2px 0px rgba(0, 0, 0, 0.30);
+  --shadow-interactive-focus-visible: 0px 0px 0px 1px rgba(0, 0, 0, 0.15), 0px 0px 0px 4px var(--primary-300);
+
+  --border-radius-sm: 4px;
+  --border-radius-md: 8px;
+  --border-radius-lg: 12px;
+  --border-radius-xl: 24px;
+  --border-radius-full: 999px;
+}
+
+@media (prefers-color-scheme: dark) {
+  .GalleryContainer,
+  .GalleryMenu,
+  .GalleryPopper,
+  .GalleryPopup,
+  .GallerySelect-listbox {
+    --font-color-soft: var(--grey-300);
+    --font-color-hard: var(--grey-50);
+    --font-color-onDark: #fff;  
+    --font-color-interactive: var(--primary-300);    
+
+    --background-surface-default: var(--grey-900);
+    --background-surface-soft: var(--grey-800);
+    --background-surface-hard: var(--grey-700);
+    --background-surface-highlight: var(--primary-500);
+    --background-surface-overlay: rgba(0,0,0,0.6);
+
+    --background-interactive-button-base: var(--primary-500);
+    --background-interactive-button-hover: var(--primary-600);
+    --background-interactive-button-active: var(--primary-700);
+    --background-interactive-button-disabled: var(--grey-800);
+
+    --background-interactive-input-base: var(--grey-900);
+    --background-interactive-input-hover: var(--grey-800);
+    --background-interactive-input-active: var(--grey-700);
+    --background-interactive-input-disabled: var(--grey-800);
+    --background-interactive-thumb: var(--grey-50);
+  
+    --border-default: 1px solid var(--grey-700);
+    --border-soft: 1px solid var(--grey-700);
+    --border-hard: 1px solid var(--grey-600);
+    --border-interactive: 1px solid var(--primary-500);
+    --border-transparent:  1px solid rgba(0,0,0,0);
+  
+    --shadow-elevation-level-1: 0px 2px 4px -0.5px rgba(0, 0, 0, 0.40);
+    --shadow-elevation-level-2: 0px 2px 8px 0px rgba(0, 0, 0, 0.30), 0px 2px 4px -0.5px rgba(0, 0, 0, 0.40);
+    --shadow-elevation-level-3: 0px 4px 8px 0px rgba(0, 0, 0, 0.32), 0px 8px 16px 0px rgba(0, 0, 0, 0.32);
+    --shadow-elevation-level-4: 0px 2px 24px 0px rgba(0, 0, 0, 0.32), 0px 16px 32px 0px rgba(0, 0, 0, 0.32);
+  
+    --shadow-interactive-button: 0px -1.5px 1px 0px var(--primary-700) inset, 0px 1.5px 1px 0px rgba(255, 255, 255, 0.2) inset, 0px 2px 4px 0px rgba(0, 0, 0, 0.25);
+    --shadow-interactive-input: 0px 2px 2px 0px rgba(0, 0, 0, 0.35);
+    --shadow-interactive-switch-track: 0px 1px 1px 0px rgba(0, 0, 0, 0.35) inset;
+    --shadow-interactive-switch-thumb: 0px 1px 2px 0px rgba(0, 0, 0, 0.30);
+    --shadow-interactive-focus-visible: 0px 0px 0px 1px rgba(0, 0, 0, 0.25), 0px 0px 0px 4px var(--primary-600);
+  }
+}
+
+.GalleryContainer {
+  background-color: var(--background-surface-default);
 }
 
 .GalleryBadge {
+  font-size: var(--caption-font-size);
+  font-family: var(--caption-font-family);
+  line-height: var(--caption-line-height);
```

Comment:
I think this can be grouped to shorten the usage:

```css
// create a new token above
--typography-caption: var(--caption-font-size) / var(--caption-line-height) var(--caption-font-family);

.GalleryBadge {
  font: var(--typography-caption);
}
```